internal/kafka/internal/presenters/admin_kafka.go

```diff
@@ -40,7 +40,7 @@ func PresentKafkaRequestAdminEndpoint(kafkaRequest *dbapi.KafkaRequest, accountS
 		Region:                     kafkaRequest.Region,
 		Owner:                      kafkaRequest.Owner,
 		Name:                       kafkaRequest.Name,
-		BootstrapServerHost:        setBootstrapServerHost(kafkaRequest.BootstrapServerHost),
+		BootstrapServerHost:        setBootstrapServerHost(kafkaRequest),
```

Comment:
@JameelB @pawelpaszki @MikeEdgar do we want to exclude the bootstrap server host from the admin endpoints as well, just like it is done in this PR? This seems consistent but I just want to be sure if this is something we want to do

Reply:
If we drop the bootstrap_server_host in the public endpoints, I think it make sense to also drop it in the admin endpoint for consistency. Unless there's strong opinions against doing that. 

Note that I have added a topic in our tech discussion if we really want to exclude both these fields from the public/admin APIs.